image_services/opera/make_opera_services.py

```diff
@@ -10,32 +10,20 @@
 from typing import List
 
 import arcpy
-import asf_search
 from arcgis.gis.server import Server
 from lxml import etree
 from osgeo import gdal, osr
 from tenacity import Retrying, before_sleep_log, stop_after_attempt, wait_fixed
 
+gdal.UseExceptions()
+gdal.SetConfigOption('GDAL_DISABLE_READDIR_ON_OPEN', 'EMPTY_DIR')
 
-def get_rasters():
-    # TODO: get rid of or update the interesects with option
-    options = {
-        'intersectsWith': 'POLYGON((-78.5937 37.5232,-74.494 37.5232,-74.494 39.8807,-78.5937 39.8807,-78.5937 '
-                          '37.5232))',
-        'dataset': 'OPERA-S1',
-        'start': '2023-10-25T08:00:00Z',
-        'processingLevel': 'RTC',
-        'polarization': 'VH',
-    }
-    results = asf_search.search(**options)
 
-    urls = []
-    for result in results:
-        for item in result.umm['RelatedUrls']:
-            url = item['URL']
-            if url.startswith('s3://') and url.endswith('VH.tif'):
-                urls.append(url.replace('s3://', '/vsis3/'))
-    return urls
+def get_rasters(dataset_name):
+    filename = f'{dataset_name}_vsis3_urls.csv'
+    with open(filename, newline='') as urlfile:
+        records = urlfile.read().split('\n')[:-1]
+    return [f'{record[:-1]}' for record in records]
```

Comment:
What character are you stripping here?

Reply:
@jhkennedy an empty space :/ Kinda hacky. I don't know why it keeps putting it there?

Reply:
@jacquelynsmale is it a space or a carriage return? If there's a space in the file that's why it'd be there, but windows does line endings with a `\r\n` (carriage return and new line), the with block has `newline=''` which does not translate `\r\n` into a single `\n` character, so your split would mean there's potentially a trailing `\r` if the csv was written on a windows box

Reply:
@jacquelynsmale if you're stripping whitespace from the end of the line, you could also just do:
```python
    return [record.rstrip() for record in records]
```
Or from both ends with `strip()`